.github/workflows/flowzone.yml

```diff
@@ -283,6 +283,24 @@ jobs:
         with:
           cmd: bash -c "echo $INPUT | tr -d '[:space:]'"
           separator: ","
+      - name: Check external contributor limitations
+        if: github.event.pull_request.head.repo.fork
```

Comment:
@klutchell this will trip users who want to fork and run Flowzone independently from us (since it will always be true).

Reply:
Good catch